WooCommerce/src/test/kotlin/com/woocommerce/android/ui/feedback/SurveyTypeTest.kt

```diff
@@ -16,16 +17,27 @@ class SurveyTypeTest {
 
     @Test
     fun `Product SurveyType url should include a milestone tag`() {
-        assertThat(PRODUCT.url.contains(Regex("product-milestone=\\d(?!\\S)"))).isTrue()
+        assertThat(PRODUCT.url.contains(Regex("product-milestone=$anyDigitAndNothingAfter"))).isTrue()
     }
 
     @Test
     fun `ShippingLabels SurveyType url should include a milestone tag`() {
-        assertThat(SHIPPING_LABELS.url.contains(Regex("shipping_label_milestone=\\d(?!\\S)"))).isTrue()
+        assertThat(SHIPPING_LABELS.url.contains(Regex("shipping_label_milestone=$anyDigitAndNothingAfter"))).isTrue()
     }
 
     @Test
     fun `Main SurveyType url should NOT include a milestone tag`() {
-        assertThat(MAIN.url.contains(Regex("milestone=\\d(?!\\S)"))).isFalse()
+        assertThat(MAIN.url.contains(Regex("milestone=$anyDigitAndNothingAfter"))).isFalse()
+    }
+
+    @Test
+    fun `SurveyType url should include app version form tag for any URL`() {
+        SurveyType.values().forEach {
+            assertThat(it.url.contains("app-version=${BuildConfig.VERSION_NAME}")).isTrue()
+        }
```

Comment:
We might want to consider using `allSatisfy` since it can provide an error message with more context: 

```
java.lang.AssertionError: 
Expecting all elements of:
  <[PRODUCT, SHIPPING_LABELS, MAIN]>
to satisfy given requirements, but these elements did not:

  <PRODUCT> expected:<[tru]e> but was:<[fals]e>

  <SHIPPING_LABELS> expected:<[tru]e> but was:<[fals]e>

  <MAIN> expected:<[tru]e> but was:<[fals]e>
```

```suggestion
        assertThat(SurveyType.values()).allSatisfy { surveyType ->
            assertThat(surveyType.url.contains("app-version=${BuildConfig.VERSION_NAME}")).isTrue()
        }
```

Reply:
That's cool idea, thanks!